### PR TITLE
commit

### PR DESCRIPTION
**Pull request by Lakshmi-kande**

### CODE REVIEW

File: __tests__/bus.test.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        busNum: '1234',
+        busType: 'AC',
+        startCity: 'Mumbai',
+        destination: 'Pune',
+        totalSeats: '50',
+        availableSeats: '50'
+      })
+      expect(res.statusCode).toBe(201);
+
+    expect(res.body).toHaveProperty('_id');
```
Comment (noobj-nimverge) [inline __tests__/bus.test.js]: did we mock db layer?

**Comment by noobj-nimverge** [comment]
Overall nice tests ...kudos to that.

Please change PR description from `commit` to something more meaningful

File: controllers/busController.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+});
+
+//@desc Create New Bus Details
+//@route POST /api/buses
+//@access private
+const createBusDetails = asyncHandler(async (req, res) => {
+    const {
+        busNum,
+        busType,
+        startCity,
```
Comment (Dincz) [inline controllers/busController.js:20]: Wouldn't **departure**  be simpler and meaningful?

File: swaggerPath.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+ *         description: ID of the bus to delete
+ *     responses:
+ *       200:
+ *         description: The bus was successfully deleted
+ *       404:
+ *         description: The bus with the specified ID was not found
+ */
+
+
+
```
Comment (Dincz) [inline swaggerPath.js:251]: Excellent documentation, testing, and coding formats,
Can restructure the file structure

**Review by noobj-nimverge — APPROVED** [review]

**Review by Dincz — APPROVED** [review]

**Pull request merged**